### PR TITLE
3.5 -- kill AQL query after timeout (#10363)

### PR DESCRIPTION
**Pull request by ObiWahn**
backport

### CODE REVIEW

**Comment by ObiWahn** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/7130/

File: CHANGELOG [hunk 1]
```diff
@@ -1,6 +1,9 @@
 v3.5.3 (XXXX-XX-XX)
 -------------------
 
+* Added new timeout option for queries. If a query does finish execution within
```
Comment (jsteemann) [inline CHANGELOG]:
```suggestion
* Added new timeout option for queries. If a query does not finish execution within
```

**Comment by ObiWahn** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/7411/

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,10 @@
+v3.5.4 (XXXX-XX-XX)
+-------------------
+
+* Added new maxRuntime option for queries. If a query does not finish execution within
+  the given time in second it will be killed.
```
Comment (jsteemann) [inline CHANGELOG]:
```suggestion
  the given time (in seconds) it will be killed.
```

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/7818/

**Comment by KVS85** [comment]
Tests blue.

**Pull request merged**